_includes/tutorials/anomaly-detection/ksql/markup/dev/make-src-file.adoc

```diff
@@ -0,0 +1,5 @@
+Now that you have a series of statements that's doing the right thing, the last step is to put them into a file so that they can be used outside the CLI session. Create a file at `src/statements.sql` with the following content:
```

Comment:
Depending on what you are trying to achieve, maybe the `INSERT` statements don't below here.  Just the queries.  Like, if a user were to take this to production, they just want the table/stream/queries, but not the fake data gen pieces?

Reply:
100% agree. I had mentioned this in the last paragraph of my initial PR description.
>I'd prefer to have my test data split out of src/statements.sql. ... Left as is, I think I need to alter the "Take it to Production" section due to the test data in the src/statements.sql.
I spent a decent amount of time trying to get input.json to work (without INSERT statements), but didn't have any luck. Could very well be a newbie problem. I can dig deeper into trying to split out the sample data and reach out to SMEs, or use a statements.sql that doesn't have any sample data in the "Take it to Production" setting. Any advice here would be greatly appreciated 🙂

Reply:
@awalther28 sorry I missed the note in the description.  Just for clarification, is the concern about taking out the `INSERT` statements that the test will then fail for the production?  If so, IIUCvyou could either (a) execute but hide the INSERT or (b) skip the test for production.  Or if I misunderstood the concern, please let me know.

Reply:
The production test is okay either way. 
The concern is that the ksqlDB test runner fails when using the same data in input.json instead, hence why there are `INSERT` statements. 
My input.json was failing with the following error, I had tried numerous iterations of the file:
`localhost:anomaly-detection awalther$ docker exec ksqldb-cli ksql-test-runner -i /opt/app/test/input.json -s /opt/app/src/statements.sql -o /opt/app/test/output.json
OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.
	>>>>> Test failed: Topic suspicious_transactions. Expected <6> records but it was <0>
Actual records:`
For some reason I can't share my input.json via github.